packages/lesswrong/lib/collections/tags/views.ts

```diff
@@ -21,6 +21,17 @@ Tags.addView('allTagsAlphabetical', terms => {
 });
 ensureIndex(Tags, {deleted:1, adminOnly:1, name: 1});
 
+Tags.addView('allPagesByNewest', terms => {
+  return {
+    selector: {
+      wikiOnly: viewFieldAllowAny
+    },
+    options: {sort: {createdAt: -1}},
+  }
+});
+
+ensureIndex(Tags, {deleted:1, adminOnly:1, createdAt: -1});
```

Comment:
These indexes are wrong. It should be `{deleted: 1, adminOnly: 1, wikiOnly: 1, createdAt: 1}`

Reply:
I'm confused about these indexes and was planning to ask you about them.  I don't care about `wikiOnly`, so why does it need to be included here? Because it's in the default view that I'm overriding?